tasks/elasticsearch-template-copy.yml

```diff
@@ -0,0 +1,9 @@
+---
+
+- file: path=/etc/elasticsearch/templates state=directory owner={{ es_user }} group={{ es_group }}
+
+- name: Copy templates to elasticsearch
+  copy: src={{ item }} dest=/etc/elasticsearch/templates owner={{ es_user }} group={{ es_group }}
+  register: load_templates
```

Comment:
So this variable is set by the template-copy. It confirms we need to do a insert as they have changed.